### PR TITLE
pangram: upgrade tests from 1.4.0 to 2.0.0

### PR DESCRIPTION
**Pull request by markhobson**
Upgrade tests to 2.0.0:
https://github.com/exercism/problem-specifications/blob/master/exercises/pangram/canonical-data.json

### CODE REVIEW

File: exercises/pangram/src/test/kotlin/PangramTest.kt [hunk 1]
```diff
@@ -12,25 +12,25 @@ class PangramTest {
 
     @Ignore
     @Test
-    fun perfectLowercasePangram() {
+    fun perfectLowerCase() {
```
Comment (dector) [inline exercises/pangram/src/test/kotlin/PangramTest.kt]: Here and below. Let's use description in backticks for test names.
E.g. [`empty sentence`](https://github.com/exercism/problem-specifications/blob/master/exercises/pangram/canonical-data.json#L10) or [`perfect lower case`](https://github.com/exercism/problem-specifications/blob/master/exercises/pangram/canonical-data.json#L18).

Comment (dector) [reply exercises/pangram/src/test/kotlin/PangramTest.kt]: Will do it later.

Comment (markhobson) [reply exercises/pangram/src/test/kotlin/PangramTest.kt]: Apologies, on holiday at the moment.

Comment (dector) [reply exercises/pangram/src/test/kotlin/PangramTest.kt]: Don't worry. :)
Enjoy your vacation!

**Pull request merged**